src/unity/python/turicreate/toolkits/_mxnet_to_coreml/_layers.py

```diff
@@ -942,10 +936,10 @@ def convert_instancenorm(net, node, model, builder):
 
     Parameters
     ----------
-    network: net
+    net: network
         A mxnet network object.
 
-    layer: node
+    node: layer
```

Comment:
Thanks for fixing all of these. I'm a big believer in always trying to leave existing code in a better state than you found it.

Reply:
Thank you for pointing it out!